docs/tutorials/motor_noise_debugging.rst

```diff
@@ -0,0 +1,196 @@
+Understanding And Debugging Communication Errors From Motor Noise
+***********************************************************************
+
+Where Does Motor Noise Come From?
+======================================
+Electric motors produce torque through interactions between the motor’s permanent magnetic field and the electric current flowing through the motor’s wire windings. 
+The electric current in the motor’s windings produces a variable magnetic field which can be controlled to exert a force on the motor’s permanent magnets/magnetic field. 
+Current is directed through the motor’s windings by applying a voltage across the windings. The direction the current flows through the windings can be switched by 
+changing the voltage polarity across the windings. In a three phase brushless DC motor like those produced by Vertiq, the three windings are connected between each of the 
+three phases as seen below in figure 1. The process of directing the current through the coils of a three-phase motor is known as commutation.
+
+.. figure:: ../_static/tutorial_images/motor_noise/simplified_phase_diagram.png
+    :align: center
+
+    Figure 1: Simplified Phase Wiring Diagram
+
+However, motors are neither perfect nor closed systems. The same switching voltages that occur during commutation to spin the motor may also cause electromagnetic interference 
+(EMI) in nearby electronic systems, including those used to drive and communicate with the motor. EMI originating from an electric motor is what we call motor noise. 
+EMI can be radiated, coupled, or conducted and can occur at different frequencies and their harmonics. Therefore, it is important to understand the sources and impacts of 
+EMI when designing mitigation techniques. 
+
+This tutorial will address the mitigation of motor noise that couples to a motor’s body through stray capacitances inside the motor. Stray capacitance (also known as 
+parasitic capacitance) will exist  between two conductors that are physically close to each other and carry different potential voltages. The electric field between the 
+two conductors will store a small charge which creates an effective capacitance. Stray capacitances are typically in the picofarad range. Unfortunately, a few picofarads 
+is enough to couple a significant motor noise to the motor’s body. Noise coupled to a motor’s body can then be conducted to conduits carrying communications cables such as 
+a vehicle frame or drone arm. Motor noise may also be radiated from the motor and its coils, but the impacts of radiated noise on unmanned vehicle systems tend to be less 
+significant and the same techniques can be used to mitigate them.
+
+
+Why Are Internal ESCs More Susceptible To Noise?
+=====================================================
+Drones that utilize electric motors with integrated ESCs tend to have longer communication wire lengths than vehicles that use external ESCs. This is because external ESCs 
```

Comment:
This says "longer communication wire lengths", might be better to just say "longer communication wires"?